hysds_commons/metadata_rest_utils.py

```diff
@@ -50,15 +50,16 @@ def get_all(es_url, es_index, es_type, query=None, logger=None):
                                                       attached_headers=headers)
 
 
-def get_by_id(es_url, es_index, es_type, ident, logger=None):
-    '''
+def get_by_id(es_url, es_index, es_type, ident, safe=False, logger=None):
+    """
     Get a spec document by ID
     @param es_url: elastic search url
     @param es_index - index containing id
     @param es_type - index containing type
     @param ident - ID
+    @param safe - returns False if set to True, raises Exception if set to True
```

Comment:
you mean `raises Exception if set to False`, right?

Reply:
good catch, ill fix the comment